web/packages/teleport/src/JoinTokens/UpsertJoinTokenDialog.tsx

```diff
@@ -81,10 +81,13 @@ const availableJoinMethods: OptionJoinMethod[] = [
   'gcp',
   'oracle',
   'github',
-].map(method => ({
-  value: method as JoinMethod,
-  label: method as JoinMethod,
-}));
+  'gitlab',
+]
+  .sort()
```

Comment:
Nit: Previously, iam was selected by default and it was the first item in the select. Now it's still selected by default, but it's the fourth item, which at least to me looks a bit off.

But I don't really have a good idea what to do with this. Especially since I suppose out of the available methods this is the most common one, so selecting gcp by default could arguably be a worse choice. Idk, maybe it's just me who finds it weird that the item selected by default is not first on the list.